types/jscodeshift/test/jscodeshift-tests.ts

```diff
@@ -134,3 +134,12 @@ testUtils.defineTest(
 
 // Can define a test
 testUtils.defineInlineTest(() => {}, { opt: true }, "import test from 'test';", "import test from './test';");
+
+// Can run inline test
+testUtils.runInlineTest(
+    () => {},
+    { opt: true },
+    { path: "/", source: "import test from 'test';" },
```

Comment:
```suggestion
    { source: "import test from 'test';" },
```

If passed, path should be a full file path instead of a dir. Since it's optional, we can leave that off here.